corehq/apps/users/bulkupload.py

```diff
@@ -438,8 +438,10 @@ def is_password(password):
                         error = custom_data_validator(data)
                         if error:
                             raise UserUploadError(error)
+                        data = {k: unicode(v) for k, v in data.iteritems()}
                         user.user_data.update(data)
                     if uncategorized_data:
+                        uncategorized_data = {k: unicode(v) for k, v in uncategorized_data.iteritems()}
```

Comment:
oh, thanks. I didn't realize that. I think what I meant to do here was `str(v)`, which seems to not blow up.